### PR TITLE
Update zh.json

### PR DESCRIPTION
**Pull request by jacarandia28**

### CODE REVIEW

**Comment by Flashfyre** [comment]
Please update this so it does not remove badges

File: lang/zh.json [hunk 1]
```diff
@@ -2761,75 +2761,42 @@
...
-      "description": "May your day be free of rain.",
-      "condition": "Visit Teru Teru Bozu Pond"
-    },
-    "slide_park": {
-      "name": "Lonely Playground",
-      "description": "Playing all by myself...",
-      "condition": "Go down the slide in Sculpture Park"
+      "name": "昆虫与咖啡",
+      "condition": "在 Baddies Bar 里点一杯咖啡。"
     }
```
Comment (nekocarpet) [inline lang/zh.json:2800]:
```suggestion
    },
    "arachnophobia_nest": {
      "name": "恐蛛症",
      "description": "Deep within the nest lies the queen and her hungry children.",
      "condition": "电锯 Spider Queen 并因此受困于 Spiders' Nest "
    },
    "giant_spider": {
      "name": "The Giant Enemy Spider",
      "condition": "在 Scenic Outlook 触发蛛网事件"
    },
    "bloody_house_illusive": {
      "name": "Narrow Your Eyes",
      "description": "她假装未察觉",
      "condition": "在 Illusive Forest 的 Bloody Eye House 中装备双马尾效果 "
    },
    "cataclysm_overgrown": {
      "name": "灭世之物",
      "description": "远古巨兽的遗骸",
      "condition": "在 Overgrown City 看见 The Thing That Ended the World"
    },
    "shinto_curse_chaser": {
      "name": "复仇诅咒",
      "condition": "在 Shinto Shrine 被抓住"
    },
    "teru_teru_bozu_pond": {
      "name": "晴天小塘",
      "description": "头上别再下雨啦！",
      "condition": "拜访 Teru Teru Bozu Pond"
    },
    "slide_park": {
      "name": "一人嬉戏",
      "description": "追尾巴转圈跑",
      "condition": "玩 Sculpture Park 的滑梯"
    }
```

File: lang/zh.json [hunk 2]
```diff
@@ -3292,11 +3259,6 @@
       "name": "Li'l Itsuki",
       "description": "The past forms the future.",
       "condition": "See Itsuki's younger self in all places he appears"
-    },
-    "silent_laughter": {
-      "name": "𝐍𝐨𝐭",
-      "description": "No one is on your side.",
-      "condition": "Experience the Silent Laughter event"
     }
```
Comment (nekocarpet) [inline lang/zh.json:3262]:
```suggestion
    },
    "silent_laughter": {
      "name": "𝐍𝐨𝐭",
      "description": "没人认同你",
      "condition": "触发 Silent Laughter 事件"
    }
```

**Review by nekocarpet — CHANGES_REQUESTED** [review]
Supplemented missing lines. See suggested changes.

**Review by Flashfyre — APPROVED** [review]

**Pull request merged**